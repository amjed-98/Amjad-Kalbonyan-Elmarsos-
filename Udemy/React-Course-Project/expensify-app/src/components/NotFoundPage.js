import React from "react";
import { Link } from "react-router-dom";

const NotFoundPage = () => (
  <div>
    404! amjad - <Link to="/">Go Home</Link>
  </div>
);

export default NotFoundPage;
