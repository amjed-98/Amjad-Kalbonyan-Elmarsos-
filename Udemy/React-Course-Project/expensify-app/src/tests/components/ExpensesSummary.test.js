import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpensesSummary';

it('should correctly render ExpensesSummary with 1 expense', () => {
  const wrapper = shallow(
    <ExpenseSummary expenseCount={1} expensesTotal={235} />
  );
  expect(wrapper).toMatchSnapshot();
});

it('should correctly render ExpensesSummary with multiple expenses', () => {
  const wrapper = shallow(
    <ExpenseSummary expenseCount={32} expensesTotal={223131235} />
  );
  expect(wrapper).toMatchSnapshot();
});
