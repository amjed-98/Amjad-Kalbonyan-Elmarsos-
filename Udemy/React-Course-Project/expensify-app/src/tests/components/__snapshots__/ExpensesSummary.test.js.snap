// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    viewing 
    1
     
    expense
     totaling 
    $2.35
  </h1>
</div>
`;

exports[`should correctly render ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    viewing 
    32
     
    expenses
     totaling 
    $2,231,312.35
  </h1>
</div>
`;
